src/util/colorScale.ts

```diff
@@ -1,7 +1,25 @@
 import * as chroma from 'chroma-js';
+import IRoute from '~/models/IRoute';
 
 class ColorScale {
-    public static getColors(colorCount: number) {
+    public static getColorMap(routes: IRoute[]) {
+        const colorMap = new Map();
+
+        let totalColorCount = 0;
+        routes.forEach((route) => {
+            totalColorCount += route.routePaths.length;
+        });
+        const colors = this.getColors(totalColorCount);
+        routes.forEach((route) => {
+            const key = route.routeId;
+            const value = colors.splice(0, route.routePaths.length);
+            colorMap.set(key, value);
+        });
+
+        return colorMap;
+    }
+
+    private static getColors(colorCount: number) {
```

Comment:
I really think we would get better results from a proper range of distinct colours. They're designed from a ux point of view and from the last sprint demo/planning this is what I understood the users wanted - a limited number of well chosen colours.

Reply:
Will be implemented here: https://github.com/HSLdevcom/jore-map-ui/issues/343

Let's discuss/plan this at slack